sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/MethodOverloadsShouldBeGrouped.cs

```diff
@@ -62,6 +63,15 @@ protected override bool IsValidMemberForOverload(MemberDeclarationSyntax member)
             }
             return true;
         }
+
+        protected override bool IsStatic(MemberDeclarationSyntax member)
+        {
+            if (member is BaseMethodDeclarationSyntax declaration) //Method or Constructor
```

Comment:
[minor] comment on separate line with space after `//` (Microsoft convention followed in team)